Worker.Extension.Sql/src/SqlInputAttribute.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extension.Sql
+{
+    public sealed class SqlInputAttribute : InputBindingAttribute
+    {
+        /// <summary>
+        /// Creates an instance of the <see cref="SqlAttribute"/>, specifying the Sql attributes
+        /// the function supports.
+        /// </summary>
+        /// <param name="commandText">The text of the command.</param>
+        /// <param name="commandType">Specifies whether commandText refers to a stored procedure or SQL query string.</param>
+        /// <param name="connectionStringSetting">The name of the app setting where the SQL connection string is stored.</param>
+        public SqlInputAttribute(string commandText, System.Data.CommandType commandType, string connectionStringSetting)
```

Comment:
Do we need to set the parameters here too?

Reply:
what do you mean here too?

Reply:
Default parameters I assume - mostly for the commandType where we default it to Text in the property itself.

Reply:
Ah I was referring to settings the Parameters field but am now realizing that is not needed (getting things mixed up with the Java classes I have been writing 😅 ). 
Looking at the in proc SqlAttribute constructor, we only take in the commandText. Is there a reason we are doing it differently here?
https://github.com/Azure/azure-functions-sql-extension/blob/66a1d06a2fede766c57842d4f297438d2d500a79/src/SqlAttribute.cs#L23

Reply:
@lucyzhang929 I was looking at the samples attribute definition instead of SqlAttribute here and added them, you were right we don't need the other params here since they have getters and setters defined appropriately.

Reply:
That being said - I honestly don't know why we don't have constructors with the additional parameters to begin with. That just seems useful to have (especially if dynamically creating a binding which is supported). 

Anyways this is fine to keep consistent and we can update later on if we have a reason to.